Project.toml

```diff
@@ -1,7 +1,8 @@
 name = "QEDbase"
 uuid = "10e22c08-3ccb-4172-bfcf-7d7aa3d04d93"
 authors = ["Uwe Hernandez Acosta"]
```

Comment:
I think you would want to add all contributors to QEDbase here, don't you?

Reply:
You are right, of course. I suggest adding the persons how actually involved in the development (significant code contributions, reviews, significant maintenance) and not every contributor. However, the actual contributors (including the external) should be listed somewhere, maybe in the changelog?

Reply:
I am fine with your solution. Also with your suggestion for future releases.